www/src/pages/docs/30_Render HTML.md

```diff
@@ -0,0 +1,1071 @@
+}
+```
+
+You cannot spot the difference by looking at the rendered DOM structure, which remains the same for both solutions.
+But looking at the code, we can conclude, that the second approach will definitely delete the whole subtree of the 
+mount-point and re-create the whole `<li>`-tags even if only *one* element of the list changes.
+
+`renderEach` instead creates a specialized mount-point in order to identify elements for re-rendering.
+This mount-point compares the last version of your list with the new one on every change and applies the minimum
+necessary patches to the DOM. Assuming again that only one item changes, the untouched items may remain inside the DOM
+and only the changed item's DOM subtree needs to be deleted and recreated. You can imagine, that this is a game changer
+concerning performance!
+
+This `renderEach` implementation does the patch-determination applying the standard `equals`-method of the list's 
+type `T`. This is the reason, why is targeted to *value* objects: They are implicitly defined by their equality!
+
+Have a look at its usage in our [master detail](/examples/masterdetail/) example.
+
+#### Reactive Rendering of Lists of Entities
+
+As `List<T>` as value of a store is a common use case, where `T` is an *entity*, thus it has some stable *identity*
+lasting all changes, fritz2 offers a special application of the `renderEach`-function for this use case.
+When dealing with entities you can pass an additional parameter called `idProvider`, which is an expression that takes
+one item `T` and determines its stable identity. Armed with this information, the internal patch-determination is then
+simply based upon the id and not upon the `equals`-method anymore.
+
+```kotlin
+// define some person-entities and "store" them
+val persons = listOf(
+    Person(1, "Fritz", 42),
+    Person(2, "Caesar", 66),
+    Person(3, "Cleopatra", 50)
+)
+
+val storedPersons = storeOf(persons)
+
+render {
+    ul {
+        storedPersons.data.renderEach(Person::id) { person ->
+            //                        ^^^^^^^^^^
+            //                        provide a function to determine the stable identity of one `T`
+            li {
+                dl {
+                    dt { +"Id" }
+                    dd { +person.id.toString() }
+                    dt { +"Name" }
+                    dd { +person.name }
+                    dt { +"Age" }
+                    dd { +person.age.toString() }
+                }
+            }
+        }
+    }
+}
+```
+
+As the identity is stable, the following properties holds for the rendered items:
+- an element is only rendered *once*, so only adding or deleting items to the store will lead to changes in the UI,
+as those new elements will be added or inserted or the deleted elements will disappear from the DOM.
+- any change to an existing item will *not* change the UI anymore. 
+
+The latter is an important aspect to consider before the use of `renderEach` for entities: 
```

Comment:
I am not sure, if this makes it clear for the user, when it might be a good idea to use one or the other version. Since most people stop reading after they first find an information, that matches their problem, this might get them on the wrong path in this case.

Reply:
What do you propose then?